fenicsxprecice/adapter_core.py

```diff
@@ -29,30 +27,23 @@ class VertexType(Enum):
 
 class Vertices:
     """
-    Vertices class provides a generic skeleton for vertices. A set of vertices has a set of global IDs, local IDs and
+    Vertices class provides a generic skeleton for vertices. A set of vertices has a set of IDs and
     coordinates as defined in FEniCSx.
     """
 
     def __init__(self, vertex_type):
         self._vertex_type = vertex_type
```

Comment:
Is the vertex type still needed? I only found vertices of type `FENICSX` (did not look extremely carefully though). If all vertices are of this type I don't see a clear reason for specifying the type and we should remove it.

Reply:
You're right, the vertex type can be removed.

Reply:
This was fixed in [0d2f727](https://github.com/precice/fenicsx-adapter/pull/15/commits/0d2f7278b3b6880163c602e31f1c06ce21edb196).